policyengine_us/variables/gov/simulation/labor_supply_response/labor_supply_response.py

```diff
@@ -95,15 +95,11 @@ class income_elasticity(Variable):
 
```

Comment:
Also change unit to "/1" (substitution too)

```suggestion
        adds = ["gov.simulation.labor_supply_responses.elasticities.income"]
```